ci/Jenkinsfile.windows

```diff
@@ -65,13 +67,16 @@ pipeline {
       parallel {
         stage('Upload') {
           steps { script {
+            zip_url = s3.uploadArtifact(env.STATUS_CLIENT_7Z)
+            jenkins.setBuildDesc(Zip: zip_url)
```

Comment:
You can't do that, it will just get overridden by the next call to `jenkins.setBuildDesc()`.

Reply:
You could just look at the implementation to see what it does:
https://github.com/status-im/status-jenkins-lib/blob/8769128d18a4f7d1001eac4212280ca4c58eb559/vars/jenkins.groovy#L78-L86

Reply:
Thanks for the input here, I will change it accordingly